azurerm/resource_arm_key_vault_certificate.go

```diff
@@ -549,13 +615,31 @@ func flattenKeyVaultCertificatePolicy(input *keyvault.CertificatePolicy) []inter
 		for _, usage := range *props.KeyUsage {
 			usages = append(usages, string(usage))
 		}
+
+		sanOutput := make(map[string]interface{}, 0)
```

Comment:
can we add a list here to allow us to set this to an empty list if it's not set

```suggestion
		sanOutputs := make([]interface{}, 0)
```